app/handlers/orgHandler.ts

```diff
@@ -24,7 +24,7 @@ export const createOrg = async (
       return { orgId : orgRef.id };
     }).catch(
     async error => {
-     return { error : error} ;
+      return { error : error };
```

Comment:
Avoid format only changes

Reply:
Ok, should we put formatting changes in a separate PR next time?